### PR TITLE
BST and Basics Updates

### PR DESCRIPTION
**Pull request by Cometnest**
Added ability to check if Paragon and Auto Paragon is active on Beastlords. I also added a function in e3_Basics that allows you to send out instance invites. This mean no more manually typing out everyones names in groups and raids!

### CODE REVIEW

File: Macros/e3 Includes/e3_Basics.inc [hunk 1]
```diff
@@ -2418,3 +2419,42 @@ SUB check_Forage
...
+|----------------------------------------------------------------------------|
+|- Sends instance invites to all those in your group or raid.
+|- Install Instructions:
+|-     Add this SUB to the bottom of e3 Includes/e3_Basics.inc
+|-     Add /doevents sendInstanceInvites to SUB basics_Background_Events
+|- Usage Instructions:
+|-     Use "/bc send instance invites" on the toon in the instance
+|-     to send out instance invites.
+|----------------------------------------------------------------------------|
+#event sendInstanceInvites "<#1#> send instance invites"
```
Comment (sirhopsalot) [inline Macros/e3 Includes/e3_Basics.inc]: How about "/bc inst invite" so it matches what you put in the chat box? 

Can just add a second event line:
#event sendInstanceInvites "<#1#> inst invite"

File: Macros/e3 Includes/e3_Basics.inc [hunk 2]
```diff
@@ -2418,3 +2419,42 @@ SUB check_Forage
...
+		/declare i int local
+		/for i 1 to ${Group.Members} {
+			/say #inst invite ${Group.Member[${i}]}
+		}
+		/next i
+	} else /if (${Raid.Members} > 0) {
+		/echo Sending instance invites to all raid members.
+		/declare i int local
+		/for i 1 to ${Raid.Members} {
+			/say #inst invite ${Raid.Member[${i}]}
```
Comment (sirhopsalot) [inline Macros/e3 Includes/e3_Basics.inc]: You've tested this with a lot of raid members? It doesn't get rate limited?

Comment (sirhopsalot) [reply Macros/e3 Includes/e3_Basics.inc]: If necessary, you could build up a long string of names, then emit the #inst invite ${longStringOfNames}

**Review by sirhopsalot — CHANGES_REQUESTED** [review]

**Comment by Cometnest** [comment]
All issues should be resolved. Tested again tonight myself and all seems well. If you see anything else, let me know.

**Review by sirhopsalot — APPROVED** [review]

**Pull request merged**